File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F13.P1.cpp
#include <cstdint>
#include <cstdio>
#include <string>

// BEGIN PROBE F13.P1
std::int32_t parse_twice(const std::string& s) {
    return 2 * std::stoi(s);
}
// END PROBE F13.P1

int main() {
    std::printf("%d\n", parse_twice("21"));
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/rust/F13.P1.rs
// BEGIN PROBE F13.P1
fn parse_twice(s: &str) -> Result<i32, std::num::ParseIntError> {
    Ok(s.parse::<i32>()? * 2)
}
// END PROBE F13.P1

fn main() {
    println!("{}", parse_twice("21").unwrap());
}
